src/main/java/org/jabref/gui/mergeentries/FetchAndMergeEntry.java

```diff
@@ -76,7 +76,17 @@ public void fetchAndMerge(BibEntry entry, List<Field> fields) {
                                   })
                                   .onFailure(exception -> {
                                       LOGGER.error("Error while fetching bibliographic information", exception);
-                                      dialogService.showErrorDialogAndWait(exception);
+                                      String localMessage = exception.getCause().getLocalizedMessage();
+                                      // client error
+                                      if (localMessage.startsWith("Client")) {
+                                          dialogService.showInformationDialogAndWait(Localization.lang("Lookup DOI"), Localization.lang("No DOI data exists"));
```

Comment:
In my opinion "No DOI data was found" sounds better :)

Imagine: A user wants to fetch DOI metadata from the web; Popup was triggerd: "No DOI data exists" --> user might think DOI data does not exist, even though it indeed MIGHT exist, but JabRef could not find it.

Reply:
I will change it quickly in one hour.